src/software/simulation/physics/physics_ball.h

```diff
@@ -122,12 +125,11 @@ class PhysicsBall
     // where the ball initially became in flight
     std::optional<Point> in_flight_origin;
     double in_flight_distance_meters;
+    // friction with other objects, such as robots/wall
+    static constexpr double BALL_FRICTION = 0.0;
 
-    // These restitution and friction values are somewhat arbitrary. Because this is an
-    // "ideal" simulation, we can approximate the ball as having perfectly elastic
-    // collisions and no friction. Because we also do not generally depend on specific
-    // behaviour when the ball collides with something, getting these values to perfectly
-    // match reality isn't too important.
-    static constexpr double BALL_RESTITUTION = 1.0;
-    static constexpr double BALL_FRICTION    = 0.0;
+    // The restitution is the amount of energy retained when bouncing off walls and linear
+    // damping is friction with the ground
```

Comment:
If the _linear damping_ is just ground friction can we rename it to that?

Reply:
Hm, it behaves like ball-ground friction but isn't exactly friction so I will keep linear damping for clarity